[BE][Inductor] fix failures in test_padding.py

shunting314 opened the pull request:
Stack from [ghstack](https://github.com/ezyang/ghstack) (oldest at bottom):
* __->__ #131417

The failure only happens [internally](https://www.internalfb.com/tasks/?t=195598864) because the main block was not executed when the tests are run internally.



cc @voznesenskym @penguinwu @EikanWang @jgong5 @Guobing-Chen @XiaobingSuper @zhuhaozhe @blzheng @wenzhe-nrv @jiayisunx @ipiszy @yf225 @chenyang78 @kadeng @muchulee8 @ColinPeppler @amjames @desertfire @chauhang

Differential Revision: [D60083954](https://our.internmc.facebook.com/intern/diff/D60083954)

shunting314 committed:
[BE][Inductor] fix failures in test_padding.py

[ghstack-poisoned]

shunting314 commented:
@shunting314 has imported this pull request.  If you are a Meta employee, you can view this diff [on Phabricator](https://www.internalfb.com/diff/D60083954).

eellison commented on test/inductor/test_padding.py:
```diff
@@ -635,8 +634,10 @@ def test_pad_channels_last(self):
         self.assertTrue(in_strides == out_strides)
 
 
+if HAS_CUDA:
+    torch.set_float32_matmul_precision("high")
+    torch.set_default_device("cuda")
```
can we put this in the testSetup and then reset in testTearDown?

shunting314 committed:
Update on "[BE][Inductor] fix failures in test_padding.py"


The failure only happens [internally](https://www.internalfb.com/tasks/?t=195598864) because the main block was not executed when the tests are run internally.



cc voznesenskym penguinwu EikanWang jgong5 Guobing-Chen XiaobingSuper zhuhaozhe blzheng wenzhe-nrv jiayisunx ipiszy yf225 chenyang78 kadeng muchulee8 ColinPeppler amjames desertfire chauhang

Differential Revision: [D60083954](https://our.internmc.facebook.com/intern/diff/D60083954)

[ghstack-poisoned]

shunting314 commented:
@shunting314 has imported this pull request.  If you are a Meta employee, you can view this diff [on Phabricator](https://www.internalfb.com/diff/D60083954).

shunting314 commented:
@pytorchbot merge

shunting314 commented:
@pytorchbot merge
